src/components/common/ColonyActionsTable/partials/ActionDescription/ActionDescription.tsx

```diff
@@ -48,16 +48,19 @@ const ActionDescription: FC<ActionDescriptionProps> = ({
 
   return (
     <div className="flex w-full items-center gap-2 sm:gap-4">
-      <Avatar
-        className={clsx('flex-shrink-0 flex-grow-0', {
-          'overflow-hidden rounded-full skeleton': loading,
-        })}
-        size="xsm"
-        seed={walletAddress.toLowerCase()}
-        title={user?.profile?.displayName || walletAddress}
-        avatar={user?.profile?.thumbnail || user?.profile?.avatar}
-      />
-
+      {loading ? (
+        <div className="h-[26px] w-[26px] rounded-full skeleton before:rounded-full" />
+      ) : (
+        <UserAvatar
+          size={26}
+          userAddress={walletAddress}
+          userName={user?.profile?.displayName ?? undefined}
```

Comment:
Personally I don't like this fallback to undefined. I think the component should handle this case